src/tcti-tabrmd.c

```diff
@@ -641,11 +641,11 @@ Tss2_Tcti_Tabrmd_Init (TSS2_TCTI_CONTEXT *context,
     g_clear_pointer (&conf_copy, g_free);
     g_clear_error (&error);
 
-    return rc;
+    return TSS2_RC_SUCCESS;
```

Comment:
Which of the new flags prompted this change?

Reply:
It should be "return rc". Thanks.